.cirrus.yml

```diff
@@ -485,13 +479,52 @@ test_build-push_task:
       bash ../ib/build-push/test.sh
 
 
+tag_latest_images_task:
+    alias: tag_latest_images
+    name: "Tag latest built container images."
+    only_if: $CIRRUS_BRANCH == $CIRRUS_DEFAULT_BRANCH
+    skip: *ci_docs
+    gce_instance: *ibi_vm
+    env:
+      <<: *image_env
+      TOOLING_FQINS: >-
+          $REGPFX/imgts:c${IMG_SFX}
+          $REGPFX/imgobsolete:c${IMG_SFX}
+          $REGPFX/imgprune:c${IMG_SFX}
+          $REGPFX/gcsupld:c${IMG_SFX}
+          $REGPFX/get_ci_vm:c${IMG_SFX}
+          $REGPFX/orphanvms:c${IMG_SFX}
+          $REGPFX/ccia:c${IMG_SFX}
+      CIVM_FQINS: >-
+          $REGPFX/skopeo_cidev:c${IMG_SFX}
+          $REGPFX/fedora_podman:c${IMG_SFX}
+          $REGPFX/prior-fedora_podman:c${IMG_SFX}
+    script: |
+      type -P skopeo &>/dev/null || \
+          dnf install -y skopeo
+      echo "$REG_PASSWORD" | \
+          skopeo login --password-stdin --username "$REG_USERNAME" "$REGPFX"
```

Comment:
This makes me so uncomfortable. Best I can tell (from looking at other repos), Cirrus does not echo the variable expansion. Do you know if there are any guarantees that it will never do so in future?

Reply:
Nvm, let's just go with a dedicated script, it's safer all around.

Reply:
That was my first thought but it started getting really complicated really fast. I've been convincing myself that I'm OK with this as it is.